inference/generativeai/llm-workshop/llama2-7b-batching-throughput/llama2-7b-batching-throughput.ipynb

```diff
@@ -0,0 +1,1713 @@
+{
```

Comment:
<p>Add CI badges - for how to do this, see https://github.com/atqy/amazon-sagemaker-examples/blob/add-badges-contributing/CONTRIBUTING.md</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added CI Badges